rbd-target-api.py

```diff
@@ -1670,6 +1670,43 @@ def _targetinfo(target_iqn):
     }), 200
 
 
+@app.route('/api/gatewayinfo/<gateway_name>', methods=['GET'])
+@requires_restricted_auth
+def gatewayinfo(gateway_name):
+    """
+    Returns the total number of active sessions on <gateway_name>
+    **RESTRICTED**
+    Examples:
+    curl --insecure --user admin:admin -X GET
+        http://192.168.122.69:5000/api/gatewayinfo/node1
+    """
+    resp_text, resp_code = call_api([gateway_name], '_gatewayinfo', '', http_method='get')
```

Comment:
Nit: should this validate that ```gateway_name``` is a valid gateway first? or perhaps just drop the ```gateway_name``` and force the caller to query the specific gateway API they are interested in and therefore avoid the indirection?

Reply:
I've removed the indirection.